flybywiresim/a32nx
# feat(fws): Add fuel low warnings

## Summary of Changes

This adds 3 new warnings to the FWC, indicating left inner fuel low, right inner fuel low, or both.

## Screenshots (if necessary)


![image](https://user-images.githubusercontent.com/2270586/218329702-b34300e7-6c42-46a9-8f12-2bfabbe993ae.png)


## References





## Additional context



Discord username (if different from GitHub): Eearslya#7831

## Testing instructions


1. Load the aircraft and start both engines.
1. Use the EFB to load 2,885kg of fuel. This should leave approximately 752kg in both inner tanks. (Outer tank will start to slowly transfer into inner, this is normal.)
1. Apply CLB thrust to engine 1 to increase fuel usage rate.
1. Clear the "THR LEVERS NOT SET" warning if it appears.
1. Approximately 30 seconds after the left fuel tank drops below 750kg, observe a "L WING TK LO LVL" warning.
1. Follow the instructions on the warning and ensure the warning lines disappear when done. (FUEL MODE SEL currently cannot be changed and therefore cannot be cleared.)
1. Move thrust lever(s) back to idle and re-load fuel. Observe the warning clears.
1. Repeat steps 2-7 for engine 2 and right fuel tank. Warning should now be "R WING TK LO LVL".
1. Repeat steps 2-7, but apply CLB thrust to both engines. Warning should now be "L+R WING TK LO LVL".


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
